### PR TITLE
added bsd 2 clause simplified license to ament_copyright

### PR DESCRIPTION
**Pull request by flynneva**

### CODE REVIEW

**Comment by flynneva** [comment]
@dirk-thomas, bump.

this is just a simple PR to add another license to this tool.
required for one of the packages I manage to pass its CI.

could we get someone assigned to review this?

File: ament_copyright/ament_copyright/template/bsd_2clause_contributing.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Any contribution that you make to this repository will
+be under the BSD 2-Clause Simplified License, as dictated by that
+[license](https://opensource.org/licenses/BSD-2-Clause).
+
```
Comment (clalancette) [inline ament_copyright/ament_copyright/template/bsd_2clause_contributing.txt:4]: Nit: remove the empty newline at the end here.

**Review by clalancette — CHANGES_REQUESTED** [review]
One minor thing to fix, then this will be ready for CI/merge.

File: ament_copyright/ament_copyright/template/bsd_2clause_contributing.txt (hunk 1 above)
Comment (flynneva) [inline ament_copyright/ament_copyright/template/bsd_2clause_contributing.txt:4]:
```suggestion
```

**Review by clalancette — APPROVED** [review]
Thanks for iterating.  Running CI on this package next.

**Comment by clalancette** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/12291/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/7264/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/10008/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/12190/)

**Pull request merged**